src/pymovements/events/processing.py

```diff
@@ -35,20 +35,18 @@
 class EventProcessor:
     """Processes event and gaze dataframes.
 
-    Attributes
+    Parameters
```

Comment:
Shouldn't parameters come here as an addition?

`event_properties` refers to both the Parameter for the initialization, and the Attribute of the instance.

Reply:
yes it should -- will fix it